package.json

```diff
@@ -58,5 +59,10 @@
         "@pika/plugin-build-web"
       ]
     ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn docs && git add docs/*"
```

Comment:
I wonder if we should make this a bit more intelligent and also auto commit a message if dosc were generated.

We could detect `src/*` changes while ignore test files with something like:

```shell
$ git diff --name-only master..HEAD \
  | grep -Ei 'src/.*.ts' \
  | if grep -Eivq 'src/.*.test.ts'; \
      git add docs/*; \
      git commit -m 'updated docs'; \
    fi
```
This would actually somewhat intelligently check if non `.test.ts` file in `src/*` have changed since `master`. If so add and commit.

Reply:
It might make more sense in this case to leverage a bash script along the lines of `hooks/pre-commit.sh` or something to avoid package.json scripts from ballooning.